tools/formatting/format-all

```diff
@@ -6,6 +6,8 @@
 # 0 on success
 # 2 is clang-format 8 could not be found 
 
+#TODO: Needs an update?
```

Comment:
TODO

Reply:
@fsimonis Do we need an update in this file to also check the format of the files in `precice/tests`?

Reply:
This is a TODO in https://github.com/precice/precice/pull/1148.